src/drawer.js

```diff
@@ -490,12 +490,13 @@ $.Drawer.prototype = {
             return;
         }
 
+        var indexOfImage = this.viewer.world.getIndexOfItem(tiledImage) % this.debugGridColor.length;
```

Comment:
Just a minor thing here: since it's not literally the index of the image (because of the %) it would be better named something like `colorIndex`, so it's not confused later.